src/main/java/workers/APIWorker.java

```diff
@@ -21,7 +22,7 @@ public APIWorker(String newId) throws IOException {
         if (!newId.contains("TST")) {
```

Comment:
Aren't both branches of this if-else identical? I don't think u need this

Reply:
yeah i just changed this